services/core-api/app/api/incidents/response_models.py

```diff
@@ -63,7 +63,8 @@
         'mine_name': fields.String,
         'mine_region': fields.String,
         'major_mine_ind': fields.Boolean,
-        'incident_timestamp': fields.DateTime,
+        'incident_timestamp': fields.DateTime(dt_format='iso8601'),
```

Comment:
Turns out this is all you need to do to use a timestamp with an offset (that and the parsing change). We're already using it with permits.